### ISSUE TITLE
Specflow plugin doesn't work on Rider 2022.3.2

### ISSUE THREAD

**Issue by IgnatievaV** [SpecFlowOSS/SpecFlow.Rider#192]
**What steps will reproduce the issue?**

Kubuntu 18

Installed Rider 2022.3.2 as another instance along with existing version 2022.1.1.

dotnet sdk: 6.0.201

Installed Specflow plugin version 1.15.3

Created a new solution with project template of Specflow.

Built and run the tests.

**What is the expected result?**

The editor shall have syntax highlighting as usual for the feature file.

When adding new step-line in the feature like "And try new step" there shall be a proposal to generate code for the new step.

**What happens instead?**

None of the expected above.

Editor just behaves as plain text editor for feature files.

(When opened same project by the other instance of Rider v 2022.1.1 which has Specflow v 1.13 - all works as expected. No issue there)

**Attachments:**

Attached screenshots of

good behavior with old Rider 2022.1.1
![image](https://user-images.githubusercontent.com/76729590/218454801-eb24a527-19e8-4c46-80f1-b020b40eb6e0.png)

bad behavior with new Rider 2022.3.2
![image](https://user-images.githubusercontent.com/76729590/218454885-39434385-3f1f-4896-ab41-2d37907461d7.png)
*.feature was missing. Reinstallation of Rider as well as the Specflow plugin didn't solve that missing mapping.
It works well after adding it manually. 
![image](https://user-images.githubusercontent.com/76729590/218455078-00a79bbe-3a09-4830-ad0e-24a029835cd4.png)

JetBrains Rider 2022.3.2

Build #RD-223.8617.53, built on January 25, 2023
Runtime version: 17.0.5+1-b653.25 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
Linux 4.15.0-43-generic
.NET 7.0.0 (Server GC)
GC: G1 Young Generation, G1 Old Generation
Memory: 1500M
Cores: 16
Registry:
    indexing.enable.entity.provider.based.indexing=false
    rdclient.asyncActions=false
    vcs.empty.toolwindow.show=false
    database.show.search.tab=false
    ide.new.project.model.index.case.sensitivity=true
    platform.projectModel.workspace.model.file.index=false
    ide.workspace.model.rbs.as.tree=false
Non-Bundled Plugins:
    com.settler.rider.specflow (1.15.3)
Current Desktop: KDE

[RIDER-89369](https://youtrack.jetbrains.com/issue/RIDER-89369/Specflow-plugin-doesnt-work-on-Rider-2022.3.2)

**Comment by seancallinan** [comment]
Same on 2023.1.2

**Comment by SimonSchwendele** [comment]
> Same on 2023.1.2

I bumped some Stuff and it should now work again.
I use it right now on 2023.2-EAP4.

They broke some stuff but it might work on the latest stable rider as well ( didnt test it ).

You can download and install the zip file [of my Fork](https://github.com/SimonSchwendele/SpecFlow.Rider/suites/13564894295/artifacts/746769721) if you like to give it a try yourself

**Comment by pluggy** [comment]
I am having the same problem with Rider 2023.3.1 on Windows, latest Specflow plugin version, .NET 4.8 and NUnit. I noticed that the *.feature file type was not bound to the Specflow plugin in Rider for some weird reason, so I added the binding. That fixed syntax highlighting and I can also create steps. But the navigation from steps to code and back does not work, and the display of failed test results does not work quite as nicely documented. also, there are no icons in the feature file that allow me to run tests quickly.

I created a new Specflow project with NUnit / .NET 6 and it worked better. now I can run unit tests from the feature file, but navigation stil does not work. Also, I have no idea what I have to do to make the Specflow Rider plugin work properly in my real solution.
.
